PoseInformationParameters information_parameters
---
string error
ram_msgs/AdditiveManufacturingPose[] selection
